### DISCUSSION TITLE
Control placement advice

### DISCUSSION THREAD

**Discussion by mattjoyce** [shanteacontrols/OpenDeck#141] [Q&A] [answered]
What is the ideal knob placement, specifically rotary pots, and distance between them?
@paradajz have you settled on a standard for your designs?

**Comment by paradajz** [comment, marked as answer]
Well... whatever suits you? On older DubFocus controllers and my current touchscreen controllers, the horizontal distance is 40mm (center to center) and vertical 25mm. On newer DubFocus controllers, horizontal distance is 29mm.

**Comment by mattjoyce** [comment]
Yes of course, personal preference is all that really matters, I was just curious to learn from others, with more experience.
I have been prototyping with paper card and 3d printer.   My ortholinear arrangement was too close for my fat thumbs, also to close to the edge.   I tried a staggered version with ever other column was offset by half a control, that was quite nice, but not sure it would translate to usefulness in practice.   I'm definitely going prototype more, and thank you for sharing your discovered optimum placing.
